sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/django.yaml

```diff
@@ -37,11 +37,11 @@ provider:
             patternAround: "(?i)\\bdjango"
         post:
           patternNot:
-            "(?i)\
-            sample|\
-            example|\
-            x{4,}|\
-            0{4,}"
+            - "(?i)\
+              sample|\
+              example|\
+              x{4,}|\
+              0{4,}"
```

Comment:
As we have the grouping in the beginning `(?i)`, I could not split the pattern that easily. If we drop the group we could benefit from the list aswell.
This applies to some of the other secrets.

Reply:
```suggestion
            - "(?i)(ex|s)ample"
            - "(\\w)\\1{4,}"
```